tools/build.sh

```diff
@@ -44,6 +37,8 @@ for DIR in src/request/*/ ; do
         cat $DIR/$url >> dist/request/$REQUEST/$JS_BUNDLE
     done
 
+    sed -i -e "s/CONFIG.ALLOWED_ORIGIN/$ALLOWED_ORIGIN/g" dist/request/$REQUEST/$JS_BUNDLE
```

Comment:
Where does `$ALLOWED_ORIGIN` come from? It's not defined anywhere. Or is it an env variable?

In that case you need to include instructions on how to set this up.

Reply:
There's also no check here if it's set, so currently if not, it replaces everything with emptiness :(